hera_mc/cm_hookup.py

```diff
@@ -446,17 +483,22 @@ def __check_hookup_cache_file_date(self, contemporaneous_minutes=5.0):
             raise
         result = self.session.query(cm_transfer.CMVersion).order_by(cm_transfer.CMVersion.update_time).all()
         cm_hash_time = Time(result[-1].update_time, format='gps')
-        file_mod_time = Time(stats.st_mtime, format='unix')
+        file_mod_time = Time(stats.st_ctime, format='unix')
```

Comment:
We're changing from looking at file modification time to creation time here. Is that really what we want? Modification time does seem like the more relevant quantity. And all of the variable names and docstrings still refer to this quantity as the "modification" time.